src/Models/Services/Utils.ts

```diff
@@ -831,3 +832,30 @@ export const getMockTimeSeriesDataArrayInLocalTime = (
         })).sort((a, b) => (a.timestamp as number) - (b.timestamp as number));
     });
 };
+
+export function formatNumber(val: number) {
+    if (Math.abs(val) < 1000000) {
+        if (Math.abs(val) < 0.000001) {
+            // display as scientific notation if greater than 100 thousandths
+            return d3.format('.1n')(val);
+        } else {
```

Comment:
if the value is 0.000001 this is how it looks like: is this what we want? should it be like using e-6 notation instead of ellipsis?
![image](https://user-images.githubusercontent.com/45217314/206566608-cffaaaea-41f2-4a25-a338-8bdc335b8d85.png)